### PR TITLE
Updated integration

### PR DESCRIPTION
**Pull request by AbrahamAjuelos**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by daryakoval** [comment]
Hi @AbrahamAjuelos , Thanks for the contribution, Before I am starting the review, please run `demisto-sdk format Packs/Darktrace` in order to format all new added files. 
Thanks.

**Comment by efelmandar** [comment]
Hi @AbrahamAjuelos, thank you for your contribution and for giving us the time to review your PR, I've now finished reviewing your changes and everything is ok. let's wait for Darys to complete her review as well and then we will be ready to schedule a demo

File: Packs/Darktrace/ReleaseNotes/2_0_0.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+##### DarktraceAIAs
+
+- Fetches AI Analyst incident events from a specified Darktrace instance
+- New Command: !darktrace-get-ai-analyst-incident-event 
+- New Command: !darktrace-get-comments-for-ai-analyst-incident-event 
+- New Command: !darktrace-post-comment-to-ai-analyst-incident-event  
+- New Command: !darktrace-acknowledge-ai-analyst-incident-event
+- New Command: !darktrace-unacknowledge-ai-analyst-incident-event
+- New Command: !darktrace-get-ai-analyst-incident-group-from-eventId
```
Comment (daryakoval) [inline Packs/Darktrace/ReleaseNotes/2_0_0.md:73]:
```suggestion
Added the following commands: 
- ***darktrace-get-ai-analyst-incident-event***
- ***darktrace-get-comments-for-ai-analyst-incident-event***
- ***darktrace-post-comment-to-ai-analyst-incident-event***
- ***darktrace-acknowledge-ai-analyst-incident-event***
- ***darktrace-unacknowledge-ai-analyst-incident-event***
- ***darktrace-get-ai-analyst-incident-group-from-eventId***
```

File: Packs/Darktrace/ReleaseNotes/2_0_0.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+##### DarktraceAdmin
+
+- New Command: !darktrace-run-advanced-search-analysis
+- New Command: !darktrace-get-device-connection-info
+- New Command: !darktrace-get-external-endpoint-details
+- New Command: !darktrace-get-similar-devices
+- New Command: !darktrace-post-to-watched-list
+- New Command: !darktrace-get-tagged-devices 
+- New Command: !darktrace-get-tags-for-device 
+- New Command: !darktrace-post-tag-to-device
```
Comment (daryakoval) [inline Packs/Darktrace/ReleaseNotes/2_0_0.md:84]:
```suggestion
Added the following commands: 
- ***darktrace-run-advanced-search-analysis***
- ***darktrace-get-device-connection-info***
- ***darktrace-get-external-endpoint-details***
- ***darktrace-get-similar-devices***
- ***darktrace-post-to-watched-list***
- ***darktrace-get-tagged-devices*** 
- ***darktrace-get-tags-for-device***
- ***darktrace-post-tag-to-device***
```

**Review by daryakoval — CHANGES_REQUESTED** [review]
Please run demisto-sdk format Packs/Darktrace in order to format all new added files.

Please fix the following unit-tests.

`test_fetch_incidents`
`test_get_ai_analyst_incident_event`
`test_get_model_breach`
`test_fetch_incidents`

**Comment by daryakoval** [comment]
Hi, 
Please handle the following validation failures: 
- [ERROR]: Packs/Darktrace/IncidentTypes/incidenttype-Darktrace_Model_Breach_Alert.json: [IT101] - The "playbookId" field is not valid - please enter a non-UUID playbook ID.
- [ERROR]: Packs/Darktrace/IncidentFields/incidentfield-Darktrace_Model_Breach_Unique_Identifier.json: [IF113] - Field name: Model Breach Unique Identifier is invalid. Field name must start with the relevant pack name.
To fix the problem, add pack name prefix to the field name. You can use the pack name or one of the prefixes found in the itemPrefix field in the pack_metadata. Example: Darktrace Model Breach Unique Identifier.
Also, make sure to update the field id and cliName accordingly. Example: cliName: darktracemodelbreachuniqueidentifier.
- [ERROR]: Packs/Darktrace/IncidentFields/incidentfield-Darktrace_Model_Breach_CommentCount.json: [IF113] - Field name: Darktreace Comment Count is invalid. Field name must start with the relevant pack name.
To fix the problem, add pack name prefix to the field name. You can use the pack name or one of the prefixes found in the itemPrefix field in the pack_metadata. Example: Darktrace Darktreace Comment Count.
Also, make sure to update the field id and cliName accordingly. Example: cliName: darktracedarktreacecommentcount.
-

**Comment by daryakoval** [comment]
Please fix the following errors: 
- [ERROR]: Packs/Darktrace/IncidentFields/incidentfield-Darktrace_Model_Breach_Unique_Identifier.json: [BA116] - cliName and id do not match.
Please change cliName to incidentmodelbreachuniqueidentifier (the flat-case version of id, excluding item-type prefixes)

- [ERROR]: Packs/Darktrace/IncidentFields/incidentfield-Darktrace_Model_Breach_CommentCount.json: [BA116] - cliName and id do not match.
Please change cliName to incidentdarktreacecommentcount (the flat-case version of id, excluding item-type prefixes)

-

**Comment by daryakoval** [comment]
You have 2 different incident fields with the same cliname `darktracecommentcount`: incidentfield-Darktrace_Model_Breach_CommentCount.json
incidentfield-Darktrace_Comment_Count.json

[ERROR]: /home/circleci/project/Packs/Darktrace/IncidentFields/incidentfield-Darktrace_Model_Breach_CommentCount.json: [GR105] - The ID 'darktracecommentcount' already exists in /home/circleci/project/Packs/Darktrace/IncidentFields/incidentfield-Darktrace_Comment_Count.json. Please update the file to have a unique ID.

[ERROR]: /home/circleci/project/Packs/Darktrace/IncidentFields/incidentfield-Darktrace_Comment_Count.json: [GR105] - The ID 'darktracecommentcount' already exists in /home/circleci/project/Packs/Darktrace/IncidentFields/incidentfield-Darktrace_Model_Breach_CommentCount.json. Please update the file to have a unique ID.

**Review by daryakoval — APPROVED** [review]

**Pull request merged**